src/pages/LearnPage.tsx

```diff
@@ -115,7 +91,7 @@ export default function LearnPage() {
         Learn
       </Typography>
 
-      <SearchBar />
+      <SearchBar value="" onChange={() => {}} onClick={() => {}} />
```

Comment:
Do these parameters have to be passed in for the Search Bar to show up? It would be nice if these parameters were left optional. I know that would require some null checks in the SearchBar component, so I'll leave that decision up to you.

Reply:
Yeah makes sense